JapaneseFilter/sections/specific.txt

```diff
@@ -111,6 +111,7 @@ xtrend.nikkei.com##.side-sticky-box[style="height: 2000px;"]
 main-hrs-game.ssl-lolipop.jp##.ads_1st
 valorantnews.jp###amgatti_sk-2
 monayoblog.com##.kaisetsu-box2
+newsweekjapan.jp##.special_bnr_area > div.bnr:first-child
```

Comment:
This should be moved to Annoyances > self-promo. Also I think `||newsweekjapan.jp/images/bnr/newsweek_ad_` will be better in this case.

Reply:
I checked your rule and it did not hide leftover on my end. What do you think about this rule: `newsweek.minkabu.jp##.bnr > a[href^="https://newsweek.minkabu.jp"]` ?

Reply:
Okay for the moment.